gcp_variant_transforms/vcf_to_bq.py

```diff
@@ -378,6 +379,53 @@ def _run_annotation_pipeline(known_args, pipeline_args):
   return annotated_vcf_pattern
 
 
+def _create_call_info_table(pipeline,  # type: beam.Pipeline
+                            pipeline_mode,  # type: PipelineModes
+                            known_args,  # type: argparse.Namespace
+                            file_path_to_file_hash  # type: Dict[str, str]
+                           ):
+  # type: (...) -> None
+  headers = pipeline_common.read_headers(pipeline,
+                                         pipeline_mode,
+                                         known_args.all_patterns)
+  _ = (headers | 'CallInfoToBigQuery' >>
+       call_info_to_bigquery.CallInfoToBigQuery(known_args.output_table,
+                                                file_path_to_file_hash,
+                                                known_args.append))
+
+
+def _write_to_bigquery_v1(pipeline,  # type: beam.Pipeline
+                          pipeline_mode,  # type: PipelineModes
+                          known_args,  # type: argparse.Namespace
+                          file_path_to_file_hash  # type: Dict[str, str]
+                         ):
+  _create_call_info_table(pipeline, pipeline_mode, known_args,
```

Comment:
Why do we need a 'sub-function' here? Will it be removed in the future?

Reply:
No, it won't be removed in the future. Instead, more functions (`_create_call_table`, `create_variant_table`, `create_variant_meta_table`, etc.) will be added into `_write_to_bigquery_v1`.